README.md

```diff
@@ -1,6 +1,6 @@
 <img align="left" width="64px" src="book/src/images/amethyst_thumb.png" />
 
-# Amethyst
+# Godot
```

Comment:
I don't think you should include unsafe blocks like this. It's not really needed here.